README.md

```diff
@@ -11,7 +11,7 @@ Installation
 
 Prerequisite - doctrine/orm
 
-`composer require sofascore/purgatory`
+`composer require sofascore/purgatory-bundle`
```

Comment:
@HypeMC hocu onda renameati i repo?

Reply:
Yeah, that's the plan, but we can do that at any time, we'll only need to change the link on packagist. I was planing on doing it after this is merged.